apps/explorer/src/app/components/price-in-market/price-in-market.spec.tsx

```diff
@@ -0,0 +1,92 @@
+import { MemoryRouter } from 'react-router-dom';
+import { MockedProvider } from '@apollo/client/testing';
+import type { MockedResponse } from '@apollo/client/testing';
+import { render } from '@testing-library/react';
+import PriceInMarket from './price-in-market';
+import { ExplorerMarketDocument } from '../links/market-link/__generated__/Market';
+
+function renderComponent(
```

Comment:
comment (non-blocking): i think this can just be a react component, if you adjust the props to be one object and use an upper case naming